### ISSUE TITLE
CLI: dependency on kbfsfuse running

### ISSUE THREAD

**Issue by philips** [keybase/client#13559]
In https://github.com/termux/termux-packages/pull/2804#issuecomment-418195668 @strib mentioned that kbfsfuse shouldn't be necessary to use git-remote-keybase. However, running this resulted in an error:

```
$ keybase git create foobar123
▶ ERROR Keybase services aren't running - KBFS client not found.
```

Which was easily solved by 

```
kbfsfuse -mount-type=none
```

cc @songgao

**Comment by strib** [comment]
You're right, though technically I was right too, since that isn't `git-remote-keybase`.  If you need to create/delete repos, you will need kbfsfuse running too.

**Comment by philips** [comment]
Ha, good point. :) 

Found one issues with `git-remote-keybase` but everything is working.

Feel free to close.

**Comment by MrCyjaneK** [comment]
It took me ages before I figured it out, it should be mentioned somewhere that it should be done to make it working on termux.

**Comment by MrCyjaneK** [comment]
If somebody is trying to mount it this is what helped me on my rooted device:
```
```▶ WARNING KBFS failed to FUSE mount at /data/data/com.termux/files/home/.config/keybase/kbfs: fusermount: exec: "fusermount": executable file not found in $PATH
▶ ERROR Mounting the filesystem failed: fusermount: exec: "fusermount": executable file not found in $PATH```
```
I got following error, so I tried to to install it... But it wasn't available in termux's `pkg` so I had to install it manually from here: https://forum.xda-developers.com/android/development/fusermount-android-rclone-mount-t3866652
And then extracted it to `$PATH` in termux.

But still it wasn't available from outside termux (but it worked inside). So this is what I've done:
Solved by using this:
`termux-setup-storage; rsync -avu --delete "/data/data/com.termux/files/home/.config/keybase/kbfs" "/data/data/com.termux/files/home/storage/shared/kbfs/"`

And finally I was able to access files from my favorite file manager. 😄.
Note: this will use double as much storage as you have in keybase, because it will copy all files from termux to sdcard, but still better than nothing.
